docs(chagelog): obs: added instances dashboard (#1119)
scaleway/docs-content: changelog/observability/december2022/2022-12-22-added-instances-dashboard.mdx

---
title: The Instances dashboard is now available in your Observability Cockpit!
status: added
author:
    fullname: 'Join the #observability-beta channel on Slack.'
    url: 'https://slack.scaleway.com/'

> date: 2022-12-15

Author: ```suggestion
date: 2022-12-22
```

category: observability
product: cockpit
---

> If you use compute [Instances](https://www.scaleway.com/en/virtual-instances/) at Scaleway Elements, you are now able to see what is going into the managed dashboard into Observability. You have access to the following key metrics:

Author: ```suggestion
The [Instances](https://www.scaleway.com/en/virtual-instances/) dashboard is available in your Observability Cockpit! You now have access to the following key metrics:
```
